frontend/apps/reader/modules/readertypography.lua

```diff
@@ -237,6 +238,7 @@ When the book's language tag is not among our presets, no specific features will
                 })
                 self.text_lang_tag = lang_tag
                 self.ui.document:setTextMainLang(lang_tag)
+                self.ui:handleEvent(Event:new("ChangedLanguage"))
```

Comment:
`ChangedTypographyLanguage` as may be someday, we'll have ChangedLanguage for dynamic UI lang switching.
Or `TypographyLanguageChanged` feels like a better event name.